argnorm/drug_categorization.py

```diff
@@ -0,0 +1,79 @@
+# These functions are run in the run function of the BaseNormalizer.
+# After ARO numbers are obtained, these functions can be executed - hence should be independent of which db is used.
+
+import pronto
+
+# Load the ArgNorm ontology from the 'aro.obo' file
+ARO = pronto.Ontology.from_obo_library('aro.obo')
+
+def get_immediate_drug_classes(aro_num: str) -> list[list]:
+    '''
+    Description: Gets the drug classes to which a gene confers resistance to.
+    Only lists the drug class column in the CARD db.
+
+    Parameters:
+        aro_num (str): ARO number. Needs to be in the form 'ARO:number'.
+
+    Returns:
+        drug_classes_list (list[list]): 
+            A two-dimensional list where each inner list represents a drug class.
+            Each inner list contains the ARO number and name of the drug class in that order. [ARO:number, name].
+    '''
+
+    GENE = ARO[aro_num]
+    confers_resistance_to_drug_class = False
+    confers_resistance_to_antibiotic = False
+
+    if confers_resistance_to_drug_class == False or confers_resistance_to_antibiotic == False:
+        for i in GENE.relationships:
+            if i.name == 'confers_resistance_to_drug_class' and confers_resistance_to_drug_class == False:
+                confers_resistance_to_drug_class = True
+
+            if i.name == 'confers_resistance_to_antibiotic' and confers_resistance_to_antibiotic == False:
+               confers_resistance_to_antibiotic = True
+
+    drug_classes = []
+
+    if confers_resistance_to_drug_class:
+        for drug_class in GENE.relationships[ARO.get_relationship('confers_resistance_to_drug_class')]:
+            drug_classes.append(drug_class)
+    
+    if confers_resistance_to_antibiotic:
+        for drug_class in GENE.relationships[ARO.get_relationship('confers_resistance_to_antibiotic')]:
+            drug_classes.append(drug_class)
+
+    drug_classes_list = []
+    for i in drug_classes:
+        drug_classes_list.append([i.id, i.name])

```

Comment:
This is unnecessary if you constructed the `drug_classes` list like this in the first place